src/test/java/net/oauth/jsontoken/AbstractJsonTokenParserTest.java

```diff
@@ -0,0 +1,324 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package net.oauth.jsontoken;
+
+import static org.junit.Assert.assertThrows;
+
+import com.google.gson.JsonParseException;
+import net.oauth.jsontoken.crypto.HmacSHA256Signer;
```

Comment:
nit: alphabetize

Reply:
Fixed here and in the other modified files.